spyder_env_manager/spyder/assets/templates/environment_info.html

```diff
@@ -0,0 +1,30 @@
+<!--
+    kernel_error.html
```

Comment:
This need to be changed to:
```suggestion
    environment_info.html
```